tests/ssl-opt.sh

```diff
@@ -8661,6 +8666,15 @@ run_test    "TLS1.3: handshake dispatch test: tls1_3 only" \
             -s "SSL - The requested feature is not available" \
             -c "SSL - The requested feature is not available"
 
+requires_openssl_tls1_3
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
+run_test    "TLS1.3: Test client hello msg work" \
+            "$O_NEXT_SRV -tls1_3 -msg" \
+            "$P_CLI min_version=tls1_3 max_version=tls1_3" \
+            1 \
+            -c "SSL - The requested feature is not available" \
+            -s "ServerHello"
+
```

Comment:
I think we should have a similar test with GnuTLS as well. Suggestion:
```
requires_gnutls_tls1_3
requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
run_test    "TLS1.3: Test client hello msg work - gnutls" \
            "$G_NEXT_SRV --priority=NORMAL:-VERS-ALL:+VERS-TLS1.3 --debug=4" \
            "$P_CLI min_version=tls1_3 max_version=tls1_3" \
            1 \
            -c "SSL - The requested feature is not available" \
            -s "SERVER HELLO was queued"
```
(Then rename the previous test with ` - openssl` for consistency.)